scripts/cddl_gen.py

```diff
@@ -1387,6 +1388,12 @@ def str_to_c_code(self, cbor_str, var_name):
         return f'uint8_t {var_name}[] = {{{", ".join(hex(c) for c in cbor_str)}}};\n'
 
 
+class xcoder_tuple(NamedTuple):
```

Comment:
Since this is now a `NamedTuple` class, we should use the [CapWords naming convention from PEP8](https://www.python.org/dev/peps/pep-0008/#class-names).

Reply:
Fixed